pages/tutorials/3.0/getting-started/compile-with-cmake.md

```diff
@@ -79,72 +103,104 @@ The first steps that need to be done are as follows (perform them in order):
-| `SFML_USE_STATIC_STD_LIBS   (Windows only)`                    | This boolean option selects the type of the C/C++ runtime library which is linked to SFML. <br>TRUE statically links the standard libraries, which means that SFML is self-contained and doesn't depend on the compiler's specific DLLs. <br>FALSE (the default) dynamically links the standard libraries, which means that SFML depends on the compiler's DLLs (msvcrxx.dll/msvcpxx.dll for Visual C++, libgcc_s_xxx-1.dll/libstdc++-6.dll for GCC). Be careful when setting this. The setting must match your own project's setting or else your application may fail to run. <br>This option should not be enabled simultaneously with BUILD_SHARED_LIBS, they are mutually exclusive.                            |
-| `SFML_GENERATE_PDB   (Visual Studio only)`                     | The boolean option controls whether Visual Studio should or shouldn't generate PDB files, which are separate files containing the debug symbols needed to debug SFML.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
-| `CMAKE_OSX_ARCHITECTURES   (macOS only)`                       | This boolean option defines which architecture SFML should be compiled for. <br>The recommended value is "x86_64" as 32-bit compilation is not supported.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
I feel like we should keep the `<br>` tags for a nicer look in the table with specific line breaks

Reply:
When viewing the rendered output, the line breaks didn't make it look better in my opinion. I assume that many of those line breaks are artifacts of the PHP->Markdown conversion rather than something intentional.

With line break:
<img width="816" alt="Screenshot 2024-10-20 at 3 32 03 PM" src="https://github.com/user-attachments/assets/26231ae6-ee0c-4c18-b6c5-5dfb667b455f">

Without line break:
<img width="1103" alt="Screenshot 2024-10-20 at 3 32 14 PM" src="https://github.com/user-attachments/assets/8bcbb408-1987-45bb-83d3-c0ed5531e28a">

Reply:
Looks better my opinion 😄

Reply:
I replaced the `<br>` tags.